bofhound/ad/models/bloodhound_user.py

```diff
@@ -8,17 +8,18 @@
 class BloodHoundUser(BloodHoundObject):
 
     COMMON_PROPERTIES = [
-        'samaccountname', 'distinguishedname', 'samaccounttype', 'objectsid',
-        'primarygroupid', 'isdeleted', 'msds-groupmsamembership',
-        'serviceprincipalname', 'useraccountcontrol', 'displayname',
+        'samaccountname', 'distinguishedname', 'isdeleted', 'msds-groupmsamembership',
+        'serviceprincipalname', 'displayname',
         'lastlogon', 'lastlogontimestamp', 'pwdlastset', 'mail', 'title',
         'homedirectory', 'description', 'userpassword', 'admincount',
         'msds-allowedtodelegateto', 'sidhistory', 'whencreated', 'unicodepwd', 'unixuserpassword',
         'domainsid', 'allowedtodelegate', 'name', 'domain', 'admincount',
         'highvalue', 'unconstraineddelegation', 'passwordnotreqd', 'enabled',
         'dontreqpreauth', 'sensitive', 'trustedtoauth', 'pwdneverexpires',
         'dontreqpreauth', 'pwdneverexpires', 'sensitive',
-        'serviceprincipalnames', 'hasspn', 'email', 'memberof'
+        'serviceprincipalnames', 'isaclprotected',
+        'hasspn', 'displayname', 'email', 'title', 'homedirectory', 'userpassword', 'unixpassword',
+        'unicodepassword', 'sfupassword', 'logonscript', 'sidhistory'
```

Comment:
I understand your point. My goal was to give a result as close as possible to SharpHound one, because the "memberof" value can be oversizing the Bloodhound GUI. I'll add this again in that case.

We could add `--gui-only` flag that only exports necessary information for BloodHound GUI. What do you think ?

Reply:
I agree, a flag to opt out of them would be a nice addition